src/DetailsView/Styles/detailsview.scss

```diff
@@ -351,11 +352,11 @@ div.insights-file-issue-details-dialog-container {
         background-color: $neutral-0;
         font-size: 14px;
         font-weight: normal;
-        border-bottom: 1px solid $neutral-alpha-8;
+        border-bottom: $detailsViewCommandBarBorderHeight solid $neutral-alpha-8;
 
         .details-view-target-page {
             margin-left: 12px;
-            padding: 13px 8px;
+            padding: 8px 8px;
```

Comment:
This looks unrelated but is important; previously, this padding was expanding the command bar to exceed its height variable, but we need the height to be statically determined for the max-height trickery in the content panes to work.